packages/php/email-editor/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "mailpoet/email-editor",
```

Comment:
Also, for review from the monorepo perspective - where do we use this package in monorepo?